lib/avo/base_action.rb

```diff
@@ -56,8 +59,10 @@ def action_name
       self.class.to_s.demodulize.underscore.humanize(keep_id_suffix: true)
     end
 
-    def initialize(model: nil, resource: nil, user: nil, view: nil)
+    def initialize(model: nil, parent_model: nil, parent_resource: nil, resource: nil, user: nil, view: nil)
```

Comment:
I'm a bit weary of adding that many things (`parent_model` and `parent_resource`) into an action. I feel they are a bit too extra.

Can we try not initializing with these two and (optionally) pass them in the `visible_in_view` method, so we have them in the Host?

Reply:
Great observation, do you think this apply to the filters too?

Reply:
Yes, definitely!

Reply:
@adrianthedev refactor done and ready for last review and merge

Reply:
sure, no problem